### ISSUE TITLE
About hell 关于地狱

### ISSUE THREAD

**Issue by ibookcc** [ImagicalCorp/ImagicalMine#520]
The dark effects of hell are sometimes not
地狱的黑暗效果时有时无

**Comment by thebigsmileXD** [comment]
*nether
And sorry, i don't understand what you mean. Light? That theres no overlay that makes your visibility darker?

**Comment by PocketKiller** [comment]
i understand you, you mean the Lava doesn't Light the level.

**Comment by thebigsmileXD** [comment]
Pe bug. Client side when the world is changed.

**Comment by robske110** [comment]
Did never see this in single player lol

**Comment by dterweij** [comment]
Please avoid foreign characters. Keep it world understandable.

**Comment by PeratX** [comment]
Please do not use Chinese!

**Comment by keithkfng** [comment]
You're allowed, but you'll need to translate them. Please try your best to avoid them though, because English is much preferred.

**Comment by TylerAndrew** [comment]
yeah i see where you guys are coming from yet google translate never seems to work for me especially translating English to french vise versa.
